src/components/main.js

```diff
@@ -70,6 +71,7 @@ class Main extends React.Component {
     // Listen for window resize
     window.addEventListener('resize', this.updateWidth);
 
+    if (this.isMobile()) this.setState({collapsed: true})
```

Comment:
You should put a handler in `this.updateWidth` as well. I noticed that if I shrink to mobile size, collapse the menu to the hamburger, and then resize to full desktop, the menu disappears completely. Think you could add something like the following to `this.updateWidth`:

```
if (this.isMobile() && !this.state.collapsed) {
  this.setState({ collapsed: true })
} else if (!this.isMobile() && this.state.collapsed) {
  this.setState({ collapsed: false })
}

Reply:
That worked well without modification. 👍